tsconfig.json

```diff
@@ -1,16 +1,17 @@
 {
   "compilerOptions": {
-    "target": "ES6",
-    "allowSyntheticDefaultImports": true,
-    "experimentalDecorators": true,
-    "esModuleInterop": true,
-    "outDir": "build",
-    "forceConsistentCasingInFileNames": true,
     "strict": true,
-    "skipLibCheck": true,
-    "moduleResolution": "Node16",
-    "module": "Node16"
+    "esModuleInterop": true,
```

Comment:
Cleaned up our main `tsconfig.json` file. Just want to make sure we have just what we really need for now. It was originally copied from an online article on how to setup a Typescript project.